### PR TITLE
Added team members as objects

### PR DESCRIPTION
**Pull request by Alexander-Lindstroem**

### CODE REVIEW

File: scripts/team.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    this.fullName = `${firstName} ${lastName}`
+    this.age = age;
+    this.location = location;
+    this.hobby = hobby;
+}
+
+let alex = new Member("Alexander", "Lindström", 31, "Sickla", "Japanese")
+let dimitar = new Member("Dimitar", "Placeholder", 1111, "Placeholder", "Placeholder")
+let mustafa = new Member("Mustafa", "Placeholder", 1111, "Placeholder", "Placeholder")
+let andre = new Member("André", "Placeholder", 1111, "Placeholder", "Placeholder")
```
Comment (andre84o) [inline scripts/team.js]: André Torabpour, 40, Råcksta, Guitar

File: scripts/team.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    this.lastName = lastName;
+    this.fullName = `${firstName} ${lastName}`
+    this.age = age;
+    this.location = location;
+    this.hobby = hobby;
+}
+
+let alex = new Member("Alexander", "Lindström", 31, "Sickla", "Japanese")
+let dimitar = new Member("Dimitar", "Placeholder", 1111, "Placeholder", "Placeholder")
+let mustafa = new Member("Mustafa", "Placeholder", 1111, "Placeholder", "Placeholder")
```
Comment (Alibaba000101) [inline scripts/team.js]: Mustafa, Larhnimi, 34, Täby, Guitar

**Review by Alibaba000101 — APPROVED** [review]

File: scripts/team.js (hunk 2 above)
Comment (DimitarIM) [reply scripts/team.js]: Dimitar, Manastirski, 21, Bromma, Workout

**Review by DimitarIM — APPROVED** [review]

**Review by andre84o — APPROVED** [review]

File: scripts/team.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    this.age = age;
+    this.location = location;
+    this.hobby = hobby;
+}
+
+let alex = new Member("Alexander", "Lindström", 31, "Sickla", "Japanese")
+let dimitar = new Member("Dimitar", "Manastirski", 21, "Bromma", "Workout")
+let mustafa = new Member("Mustafa", "Larhnimi", 34, "Täby", "Guitar")
+let andre = new Member("André", "Placeholder", 1111, "Placeholder", "Placeholder")
+let adam = new Member("Adam", "Placeholder", 1111, "Placeholder", "Placeholder")
```
Comment (AwdamnKaya) [inline scripts/team.js]: Kaya, 34, Sätra, Music

**Pull request merged**